src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -744,32 +744,32 @@ export const AutomatedAnalysisCard: React.FC<AutomatedAnalysisCardProps> = (prop
   }, [usingArchivedReport, usingCachedReport, report, isLoading, errorMessage]);
 
   return (
-    <ResizableCard
+    <DashboardCard
```

Comment:
I think we should apply some truncating styles for label texts on header.
![Screenshot from 2023-01-26 12-17-54](https://user-images.githubusercontent.com/68053619/214906935-fafb4043-0cb6-4b4c-9ea5-8e823652afb1.png)